### PR TITLE
Add toga to nuitka-watch

### PR DESCRIPTION
**Pull request by KRRT7**

### CODE REVIEW

File: cases/toga/case.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:indentation
+# yamllint disable rule:comments-indentation
+# too many spelling things, spell-checker: disable
+---
+- case: toga-latest
+  # Integer version, increased when results become invalidated by cases changes.
+  version: 1
+  filename: "TogaCase.py"
```
Comment (kayhayen) [inline cases/toga/case.yml]: I prefer names like "*Using.py"  for some reason.

File: cases/toga/TogaCase.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    button = toga.Button("Hello world", on_press=button_handler)
+    button.style.padding = 50
+    button.style.flex = 1
+    box.add(button)
+    return box
+
+def main():
+    return toga.App("First App", "org.beeware.helloworld", startup=build)
+
+if __name__ == "__main__":
```
Comment (kayhayen) [inline cases/toga/TogaCase.py]: What about the NUITKA_LAUNCH_TOKEN thing, to display it or not, that seems like  good idea for coverage of GUI when interactively ran or not.

**Review by kayhayen — APPROVED** [review]

File: cases/toga/case.yml (hunk 1 above)
Comment (KRRT7) [reply cases/toga/case.yml]: preference, or requirement?

File: cases/toga/TogaCase.py (hunk 2 above)
Comment (KRRT7) [reply cases/toga/TogaCase.py]: sure, i think it's good enough as it is though, maybe once we have the watch running well enough we can extend the GUI apps in that way

**Comment by Maxwell175** [comment]
OK

**Comment by Maxwell175** [comment]
OK

**Pull request closed without merging**